### PR TITLE
chore(deps): allow Laravel 11

### PR DESCRIPTION
**Pull request by canvural**
Support Laravel 11 alongside other versions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     },
     "scripts": {
         "test:types": "phpstan analyse --ansi --memory-limit 256M",
-        "test:unit": "phpunit --colors=always -d memory_limit=1408M",
+        "test:unit": "phpunit --colors=always -d memory_limit=2048M",
```
Comment (szepeviktor) [inline composer.json]: > Time: 00:16.143, Memory: 1.15 GB

Should we be liberal with the memory thus loose control how much is really needed?

Comment (canvural) [reply composer.json]: Yeah I think was an issue with PHPUnit version. Reverted the change and trying again now.

**Comment by szepeviktor** [comment]
@canvural How were you able to fix all 3 E2E test?

**Review by szepeviktor — APPROVED** [review]

**Comment by canvural** [comment]
> @canvural How were you able to fix all 3 E2E test?

https://github.com/larastan/larastan/commit/c5b69f76eb536b1a60865e5eb67ad6f640eddf19 and https://github.com/larastan/larastan/commit/1a570a4c8be1f9cc749286858032787a036781e6

**Pull request merged**